run_tag.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
```

Comment:
@udinobi  any reason why we are implementing new code for runner file. this was not suggested by platops test community. if there do not accept it we have to change the code back again. we have had this issues before if we do not align with their framework there are not going to accept the handover.